### PR TITLE
Draft PR for Updation of Documentation

### PR DESCRIPTION
**Pull request by adiyaar24**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/set-up-an-aws-vm-build-infrastructure.md [hunk 1]
```diff
@@ -256,7 +259,8 @@ You can configure the following settings in your `pool.yml` file. You can also l
 
 #### user data example
 
-Provide [cloud-init data](https://docs.drone.io/runner/vm/configuration/cloud-init/) in either `user_data_path` or `user_data`.
+Provide [cloud-init data](https://docs.drone.io/runner/vm/configuration/cloud-init/) in either `user_data_path` or `user_data` if you need custom configuration , below is an example for linux based 
+distribution
```
Comment (adiyaar24) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/set-up-an-aws-vm-build-infrastructure.md:263]: Please ask the Engineering team to provide a sample user data for Windows Distribution as well

As of now if you don't specify the user data it works flawlessly

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/set-up-an-aws-vm-build-infrastructure.md [hunk 2]
```diff
@@ -165,7 +166,7 @@ instances:
         access_key_id: XXXXXXXXXXXXXXXXX
         access_key_secret: XXXXXXXXXXXXXXXXXXX
         key_pair_name: XXXXX
-      ami: ami-051197ce9cbb023ea
+      ami: ami-xxx  ## Linux Amd64 AMI should be passed
```
Comment (jimsheldon) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/set-up-an-aws-vm-build-infrastructure.md:169]: In the "Launch the EC2 instance" section we say the VM must be Ubuntu. Can we specify Ubuntu here?

```suggestion
      ami: ami-xxx  ## Ubuntu Amd64 AMI must be passed
```

**Review by jimsheldon — CHANGES_REQUESTED** [review]

**Review by nofarb — APPROVED** [review]
approved

**Comment by dewan-ahmed** [comment]
Hi @adiyaar24 please update this PR to resolve conflicts and remove "draft" from title if it's good to get merged. thanks.

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**